src/processTargets/marks/CursorStage.ts

```diff
@@ -1,13 +1,10 @@
 import { Target } from "../../typings/target.types";
-import { CursorMark } from "../../typings/targetDescriptor.types";
 import { ProcessedTargetsContext } from "../../typings/Types";
 import { isReversed } from "../../util/selectionUtils";
 import { MarkStage } from "../PipelineStages.types";
 import WeakTarget from "../targets/WeakTarget";
 
 export default class CursorStage implements MarkStage {
-  constructor(_modifier: CursorMark) {}
```

Comment:
`MarkStage` doesn't require any particular constructor signature, so removing this to squash `no-empty-function`.

Reply:
See above re uniformity